scripts/new.js

```diff
@@ -0,0 +1,26 @@
+$('#write').on('mouseup keyup', function(event) {
+    event.preventDefault();
+    var prop = {};
+
+    // read values from form
+    prop.title = $('#article-title').val();
+    prop.author = $('#article-author').val();
+    prop.authorUrl = $('#article-author-url').val();
+    prop.category = $('#article-category').val();
+    prop.publishedOn = new Date();
+
+    $articleBody = $('#article-body').val();
+    prop.body = marked($articleBody);
+
+    // display preview on page
+    var newArticle = new Article(prop);
```

Comment:
Every time you have a keyup here, you're creating a new article. That could be causing some problems.